### PR TITLE
Create rsref-illinois.md

### PR DESCRIPTION
**Pull request by rsref**
test pull request

### CODE REVIEW

File: rsref-illinois.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//Places to go
+Chicago
+
+//Things to do
+
+Skiiing
```
Comment (ghost) [inline rsref-illinois.md:6]: where's the best place to ski?

**Comment by kumareshhj** [comment]
hi hi..

File: rsref-illinois.md (hunk 1 above)
Comment (rsref) [inline rsref-illinois.md:6]: Wisconsin Dells

**Comment by rsref** [comment]
Hi Kumaresh

**Comment by rsref** [comment]
Leave a comment

**Pull request closed without merging**